lib/src/bottom_sheets/cupertino_bottom_sheet.dart

```diff
@@ -264,7 +275,12 @@ class CupertinoScaffold extends StatefulWidget {
     assert(useRootNavigator != null);
     assert(enableDrag != null);
     assert(debugCheckHasMediaQuery(context));
-    assert(debugCheckHasMaterialLocalizations(context));
+    final isCupertinoApp = Theme.of(context, shadowThemeOnly: true) == null;
```

Comment:
I am not sure this is a great way to check if an app uses CupertinoApp,
You could do this and it would keep throwing the error
```dart
Theme(
   data: ThemeData(), 
   child: CupertinoApp(...),
);
```

I think using this would be better:
```dart
final hasMaterialLocalizations = Localizations.of<MaterialLocalizations>(context, MaterialLocalizations) != null;
final  barrierLabel = hasMaterialLocalizations ? MaterialLocalizations.of(context).modalBarrierDismissLabel : '';
```

Reply:
You're right. I didn't test this combination. 
I will replace my implementation with yours.